DeAceroDLabs/marketplace-frontend
# [DMP-507] Remove "0" from cart icon

## What is done

[[DMP-507] Publicar nombre de usuario en Auth0](https://deacero.atlassian.net/browse/DMP-507)

## How is it done

1. Modified the validation of shoppingCartItems length to make sure is bigger than 0.

## How to test

1. Run the app `npm start`.
2. Check that in the header there is not a "0" next to the cart icon anymore.
